CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/undo/UndoGraphEditOperation.java

```diff
@@ -716,14 +716,14 @@ public void updateUndo(UndoGraphEditState state, int instruction) {
         @Override
         public void execute(UndoGraphEditState state, GraphWriteMethods graph) {
             if (VERBOSE) {
-                System.out.println("setDoubleValueOperation.execute(" + state.getCurrentAttribute() + ", " + state.getCurrentId() + ", " + Double.longBitsToDouble((long) (Double.doubleToRawLongBits(graph.getDoubleValue(state.getCurrentAttribute(), state.getCurrentId())) ^ state.getCurrentDouble())) + ")");
+                System.out.println("setDoubleValueOperation.execute(" + state.getCurrentAttribute() + ", " + state.getCurrentId() + ", " + Double.longBitsToDouble((Double.doubleToRawLongBits(graph.getDoubleValue(state.getCurrentAttribute(), state.getCurrentId())) ^ state.getCurrentDouble())) + ")");
```

Comment:
Believe it or not but sonar reckons the cast is actually necessary (seems to think we re parsing longBitsToDouble an integer)

Reply:
I don't believe it... but sure, reverted.